tests/io/test.base.js

```diff
@@ -1,3 +1,5 @@
+import sinon from 'sinon';
```

Comment:
This could be a global sandbox instance created in the global test setup that gets automatically torn down too. See https://github.com/mozilla/addons-frontend/blob/master/tests/setup.js#L62 for an example - although as mentioned I've filed an issue re: calling this sinon (see https://github.com/mozilla/addons-frontend/issues/2546)

If you follow the example above you probably should remove the line re: format since sinon.format has been deprecated anyway.